packages/yoastseo/src/languageProcessing/helpers/image/getAltAttribute.js

```diff
@@ -0,0 +1,22 @@
+/** @module stringProcessing/getAltAttribute */
+
```

Comment:
```suggestion
```

Reply:
This line is not necessary. And also since `stringProcessing` is no longer available this module is not available.